providers/user.rb

```diff
@@ -243,7 +243,9 @@ def retrieve_db(attempt = 0)
       @new_resource.connection['host'],
       @new_resource.connection['port'],
       connect_timeout: 15,
-      slave_ok: true
+      slave_ok: true,
+      ssl_verify: false,
```

Comment:
Would you be able to make this an attribute, even if the default is `false`? If people have installed a verifiable certificate on the server, IMHO they should be able to use that to prevent against MITM attacks.